lib/omise/charge.ex

```diff
@@ -305,4 +305,26 @@ defmodule Omise.Charge do
     opts = Keyword.merge(opts, as: %Omise.Refund{})
     get("#{@endpoint}/#{id}/refunds/#{refund_id}", [], opts)
   end
+
+  @doc ~S"""
+  List all charge schedules.
+
+  Returns `{:ok, schedules}` if the request is successful, `{:error, error}` otherwise.
+
+  ## Query Parameters:
+    * `offset` - (optional, default: 0) The offset of the first record returned.
+    * `limit` - (optional, default: 20, maximum: 100) The maximum amount of records returned.
+    * `from` - (optional, default: 1970-01-01T00:00:00Z, format: ISO 8601) The UTC date and time limiting the beginning of returned records.
+    * `to` - (optional, default: current UTC Datetime, format: ISO 8601) The UTC date and time limiting the end of returned records.
+
+  ## Examples
+
+      Omise.Charge.list_schedules
+
+  """
+  @spec list_schedules(Keyword.t, Keyword.t) :: {:ok, Omise.List.t} | {:error, Omise.Error.t}
```

Comment:
Might be better to add `| none()` type to the parameter list since caller can omit it na.